src/test/kotlin/io/hsar/wh40k/combatsimulator/content/ItemDatabaseTest.kt

```diff
@@ -41,6 +36,33 @@ class ItemDatabaseTest {
         val result = objectUnderTest.itemsByItemRef.getValue(expectedWeaponRef)
 
         // Assert
-        assertThat(result, equalTo(expectedItem))
+        assertThat(result.itemRef, equalTo(expectedItem.itemRef))
+        assertThat(result.itemName, equalTo(expectedItem.itemName))
+        assertThat(result.itemType, equalTo(expectedItem.itemType))
+        assertThat(result.modifiesAttributes[WEAPON_TYPE], equalTo(expectedItem.modifiesAttributes[WEAPON_TYPE]))
+
+        // Check both expected and actual have one action available
+        assertThat(
+                (result.modifiesAttributes[ACTIONS] as ActionValue).value.size,
+                equalTo((expectedItem.modifiesAttributes[ACTIONS] as ActionValue).value.size))
+
+        // and that both expected and actual have the action as the same class of ActonOption
+        assertThat(
+                (result.modifiesAttributes[ACTIONS] as ActionValue).value[0]::class,
```

Comment:
You probably want to use [this matcher](http://hamcrest.org/JavaHamcrest/javadoc/1.3/org/hamcrest/core/IsInstanceOf.html)

Reply:
Updated to use that matcher.